### DISCUSSION TITLE
Android beta App phone going flashing/black

### DISCUSSION THREAD

**Discussion by mkorsg** [advplyr/audiobookshelf-app#92] [General]
Hey
I just started using your server docker/android app, and awesome work.

Found a problem when i download a book to the phone, when i try listening offline the phone starts flaching black and i can only see battery status on black screen when i can see anything, the sound is running but can't interact with app or anything. Only thing i can do is shootdown or restart phone. Run with no problems streaming, so thats what i will do for now :)

Phone: Sony xperia 5 II
Filetype of audiobook m4b
0.9.39-beta

/mkorsg

**Comment by TeroKeso** [comment]
I did replicate the problem. Android 11, Motorola one vision. Did get the error of file system read locks down and then the phone rebooted.

**Reply by advplyr** [reply]
What is the error message saying?

**Reply by TeroKeso** [reply]
![image](https://user-images.githubusercontent.com/29271631/151560153-122f9edb-1596-4d4e-a075-718953953fce.png)
![image](https://user-images.githubusercontent.com/29271631/151560186-ddc561e1-9343-4c8e-b87d-58f7edd34165.png)

**Reply by mkorsg** [reply]
I didn't get anything i could see like that, nice your phone accually told you something.

**Reply by advplyr** [reply]
Did you see the issue with OnePlus devices #35. I wonder if the same solution will work for you.
The solution was to remove the cover image from the audiobook and download it without the cover. Then playing that audiobook would not crash. There is a problem with locally downloaded cover images being used in the android notification that I don't understand yet.

**Reply by mkorsg** [reply]
That worked, thanks a lot for the quick help.

**Reply by mkorsg** [reply]
workaround could be to just download the audiofiles ? so no .jpg on phone

**Reply by advplyr** [reply]
What I did for the OnePlus users was just not use the cover image in the android notification. I will add the same check for your devices so you don't need to worry about whether it has a cover image or not.
I thought this was only happening for OnePlus, so I will look into this again to see if I can find a permanent solution.
